packages/native/index.d.ts

```diff
@@ -32,6 +32,11 @@ export interface UpdateRow {
   id: number;
   timestamp: Date;
   data: Buffer;
+  docId?: string;
+}
+export interface InsertRow {
+  docId?: string;
```

Comment:
remove `?` ?

Reply:
this is actually intended in current design.